src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneQuickFilter.tsx

```diff
@@ -83,6 +83,6 @@ export class SceneQuickFilter extends SceneObjectBase<SceneQuickFilterState> {
 const getStyles = () => ({
   filter: css`
     flex: 1;
-    min-width: 0;
+    min-width: 112px;
```

Comment:
Prevents the quick filter to shrink to none on narrow screens (not strictly related to this PR).